auto_submit/lib/service/validation_service.dart

```diff
@@ -334,8 +335,13 @@ Exception: ${exception.message}
     final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
     final int number = messagePullRequest.number!;
     // Pass an explicit commit message from the PR title otherwise the GitHub API will use the first commit message.
-    String commitMessage = messagePullRequest.title!;
-    commitMessage = commitMessage.replaceFirst('Revert "Revert', 'Reland'); // Cleanup auto-generated revert messages.
+    final String prTitle =
+        messagePullRequest.title!.replaceFirst('Revert "Revert', 'Reland'); // Cleanup auto-generated revert messages.
+    final String commitMessage = ('''
```

Comment:
Should this start with a newline? I thought the first line is the commit title

```suggestion
    final String commitMessage = ('''$prTitle
```

Reply:
If the first line of a multiline string consists solely of whitespace, it's ignored by the parser, including the newline: https://github.com/dart-lang/language/blob/master/specification/dartLangSpec.tex#L9112